libs/client/cornflow_client/airflow/dag_utilities.py

```diff
@@ -226,13 +226,17 @@ def cf_solve(fun, dag_name, secrets, **kwargs):
     except NoSolverException as e:
         if config.get("msg", True):
             print("No solver found !")
+        # We reconnect in case the solver has been more than 24 hours solving before the error is raised
+        client = connect_to_cornflow(secrets)
```

Comment:
Would it be better to move the connect call inside try_to_save_error and pass secrets as an argument instead of client? This way, we could avoid creating a new connect call each time it's used.

Reply:
No, because the method `try_to_save_airflow_log()` also needs the client updated to be able to save the log as well. So if we update it inside the `try_to_save_error()` it would not be updated for the other method.